pims/api.py

```diff
@@ -57,6 +57,33 @@ def raiser(*args, **kwargs):
     TiffStack = not_available("tifffile, libtiff, or PIL/Pillow")
 
 
+# As above
+from pims.image_sequence import (ImageSequence_skimage, ImageSequence_mpl,
+                                 ImageSequence_scipy, ImageSequence_pil,
+                                 ImageSequence_tifffile)
+if not pims.image_sequence.skimage_available():
+    ImageSequence_skimage = not_available('skimage')
+if not pims.image_sequence.skimage_available():
```

Comment:
I would opt for an (optional) classmethod `pkg_available()` for every reader class.